pkg/kubelet/kubelet.go

```diff
@@ -1343,6 +1362,16 @@ type Kubelet struct {
 
 	// Track node startup latencies
 	nodeStartupLatencyTracker util.NodeStartupLatencyTracker
+
+	// sends an update when node is registered.
+	nodeRegistrationCh chan struct{}
+
+	// clean up EventHandlers added to node informers.
+	cleanupNodeEventHandler func()
+
+	// whether initial static pods registration was completed or not
+	// before node became ready.
+	staticPodsRegistered bool
```

Comment:
Could you make the name more specific since static pods are not really static and can change? For example, "initialStaticPodsRegistered"

Reply:
Done.